util/util.go

```diff
@@ -49,11 +83,12 @@ func NewPeer(ctx context.Context, datastorePath string, port int, debug bool) (*
 		return nil, err
 	}
 
-	lite, err := ipfslite.New(ctx, ds, h, dht, nil)
+	peerManager.peer, err = ipfslite.New(ctx, peerManager.datastore, peerManager.host, peerManager.dht, nil)
 	if err != nil {
 		return nil, err
 	}
 
-	lite.Bootstrap(ipfslite.DefaultBootstrapPeers())
-	return lite, nil
+	peerManager.peer.Bootstrap(ipfslite.DefaultBootstrapPeers())
+
+	return peerManager, nil
```

Comment:
But here I get the error: `cannot use peerManager (variable of type manager) as server.PeerManager value in return statement: missing method Peer`

Reply:
Ok I fixed this problem by not using a pointer when extending the `manager` type to conform to the `PeerManager` interface. This:

```
func (m manager) Peer() *ipfslite.Peer {
	return m.peer
}
```

vs

```
func (m *manager) Peer() *ipfslite.Peer {
	return m.peer
}
```